src/main/java/com/minecolonies/core/colony/buildings/workerbuildings/BuildingCook.java

```diff
@@ -180,7 +177,7 @@ public BlockPos getNextSittingPosition()
      */
     public boolean getIsCooking()
     {
-        final ICitizenData citizen = this.getModuleMatching(CraftingWorkerBuildingModule.class, m -> m.getJobEntry() == ModJobs.cookassistant.get()).getFirstCitizen();
+        final ICitizenData citizen = this.getModule(BuildingModules.COOKASSISTENT_WORK).getFirstCitizen();
         return citizen != null && isCooking && isCookingTimeout > 0;
     }
```

Comment:
Just to have conflict chunks with my PR where I get rid of this! =D

Reply:
hihi